### PR TITLE
Update Moonbeam ledger guide for ledger live integration

### PR DESCRIPTION
**Pull request by eshaben**
### Description

This PR adds a new section to the Moonbeam Ledger live docs that covers the new integration with Ledger Live, enabling users to send and receive GLMR using Ledger Live and their Ledger device.

The lighting was hard to work with where I'm staying. So I struggled with the images, and kind of got to a point where I said it is what it is 🤷‍♀️ 

### Checklist

- [x] If this requires translations for the `moonbeam-docs-cn` repo, I have created a ticket for the translations in Jira
- [ ] If pages have been moved around, I have created an additional PR in `moonbeam-mkdocs` to update redirects
- [ ] If pages have been moved around, I have run the `move-pages.py` script to move the pages and update the image paths on the chinese repo
    - [ ] After the script has been run, I have created an additional PR in `moonbeam-docs-cn`
- [x] If images have been added, I have run the `compress-images.py` script to compress the images.
- [ ] If variables (in variables.yml) need to be updated (such as a name change), I have updated the `moonbeam-docs-cn` repo to use the new variables
- [ ] If this page requires a disclaimer, I have added one

### Corresponding PRs

N/A

### After Translation Requirements

- [ ] Will need to create PR in `moonbeam-docs` repo to remove images
- [ ] Will need to create PR in `moonbeam-docs` repo to remove variables
- [ ] Will need to create PR in `moonbeam-mkdocs` repo to add redirects for Chinese site
- [x] No additional PRs are required after the translations are done

#### Items to be Updated

N/A

### CODE REVIEW

**Review by albertov19 — APPROVED** [review]
LGTM, I would include this as a snippet in both Ethereum and Moonbeam pages :)

File: tokens/connect/ledger/ethereum.md [hunk 1]
```diff
@@ -95,4 +96,6 @@ And that is it! You've signed a transaction and sent some tokens on Moonbeam usi
 
 ![MetaMask Ledger Allow Contracts Tx](/images/tokens/connect/ledger/ethereum/ledger-11.png)
 
+--8<-- 'text/ledger/ledger-live.md'
```
Comment (wuxinxin2000) [inline tokens/connect/ledger/ethereum.md:99]: I think this line is missing in PureStake/moonbeam-docs-cn repository PR#202.

Comment (wuxinxin2000) [inline tokens/connect/ledger/ethereum.md:99]: I think this line is missing in PureStake/moonbeam-docs-cn repository PR#202.

Comment (wuxinxin2000) [inline tokens/connect/ledger/ethereum.md:99]: I think this line is missing in PureStake/moonbeam-docs-cn repository PR#202.

File: tokens/connect/ledger/moonbeam.md [hunk 2]
```diff
@@ -92,4 +98,6 @@ And that is it! You've signed a transaction and sent some GLMR tokens using your
 
 ![MetaMask Ledger Allow Contracts Tx](/images/tokens/connect/ledger/moonbeam/ledger-11.png)
 
---8<-- 'text/disclaimers/third-party-content.md'
\ No newline at end of file
+--8<-- 'text/ledger/ledger-live.md'
```
Comment (wuxinxin2000) [inline tokens/connect/ledger/moonbeam.md:101]: I think this line is also missing in corresponding file in PureStake/moonbeam-docs-cn repository PR#202.

**Pull request merged**